src/components/Home/NewPlayer.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import {useState} from 'react';
+import {Button, Input} from '@material-ui/core';
+
+interface IProps {
+	handleSubmit: Function;
```

Comment:
I think it might be useful to specify function singnature: 
```
handleSubmit: (name: string) => void;
```

Reply:
Fixed